Documentation/subcommands/config.md

```diff
@@ -0,0 +1,46 @@
+# rkt config
+
+This command prints the rkt configuration per stage in JSON format.
+
+## Example
+
+```
+$ rkt config
+{
+  "stage0": [
```

Comment:
Addressed OOB notes. I could not reproduce the oauth/basic mutual incompatibility, I mixed up the already documented override semantics of system/user directories.